main.py

```diff
@@ -113,9 +92,17 @@ def main(input_path, output_path):
         return
 
     # Create output directory
+    output_folder_path = None
+    
     if output_path:
-        create_directory(output_path)
-        logging.debug("Creating Output Path {}".format(output_path))
+        output_folder_path = output_path
+
+    else:
+        output_folder_path = 'output'
+        logging.debug("Output path not specified")
```

Comment:
If the user doesn't provide the output directory, the output should be printed to the console itself. 
@jiteshm17 Can you make the changes accordingly?

Reply:
I was thinking that maybe it would be a good idea to save them in a temp directory instead of the console? Like if the file is very big, it might not seem appropriate and might even be difficult to copy and paste later.

What do you think @prabhakar267 ?